assets/erc-7635/contracts/interfaces/IERC721Receiver.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
We prefer `CC0-1.0` for reference implementations, but allow most permissive non-copyleft licenses (so `MIT` is fine, but `GPL` is not.)

If you have the authority to re-license this code, could you change it to `CC0-1.0` or some other non-copyleft option?